script.js

```diff
@@ -0,0 +1,79 @@
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
we don't use DOMContentLoaded, 
just put the <script> tag at the bottom of the body